src/geometry/CTiglTransformation.h

```diff
@@ -125,6 +128,17 @@ class CTiglTransformation
     // Getter for matrix values
     TIGL_EXPORT double GetValue(int row, int col) const;
 
+    // Set rotation and scaling to trivial values,
+    // the affine part will be the identity,
+    // the translation part will remain unchanged.
+    TIGL_EXPORT void SetTrivialScalingAndRotation();
```

Comment:
This function is very specific and it is also not clear for the user, that it actually does!

Possibilities could be 
 - Reset -> Identity transformation
 - and then SetTranslation(...)

Reply:
Ok, I will implement a function "GetTranslation()" and use it instead or reset the rotation and scaling values to their trivial values. It will be more clear ^^

Reply:
done
commit: c2d9b9084f9cf34caa645b4b95a536bd62a6541b